apps/studio/components/interfaces/Organization/GeneralSettings/AIOptInLevelSelector.tsx

```diff
@@ -0,0 +1,88 @@
+import { ReactNode } from 'react'
+import { Control } from 'react-hook-form'
+import { cn, FormField_Shadcn_, RadioGroup_Shadcn_, RadioGroupItem_Shadcn_ } from 'ui'
+import { FormItemLayout } from 'ui-patterns/form/FormItemLayout/FormItemLayout'
+import OptInToOpenAIToggle from './OptInToOpenAIToggle'
+import { AIOptInFormValues } from 'hooks/forms/useAIOptInForm'
+
+interface AIOptInLevelSelectorProps {
+  control: Control<AIOptInFormValues>
+  disabled?: boolean
+  label?: ReactNode
+  layout?: 'horizontal' | 'vertical' | 'flex-row-reverse'
+}
+
+export const AIOptInLevelSelector = ({
+  control,
+  disabled,
+  label,
+  layout = 'vertical',
+}: AIOptInLevelSelectorProps) => {
+  const content = (
+    <div className="space-y-4">
+      <FormField_Shadcn_
+        control={control as any}
```

Comment:
hmm is the `any` here necessary? i'd like to avoid using `any` as much as possible